core/templates/pages/blog-dashboard-page/blog-dashboard-page.component.spec.ts

```diff
@@ -254,11 +254,12 @@ describe('Blog Dashboard Page Component', () => {
   it('should display alert when unable to fetch blog dashboard data',
     fakeAsync(() => {
       spyOn(loaderService, 'showLoadingScreen');
+      spyOn(component, 'getUserInfoAsync').and.returnValue(Promise.resolve());
       spyOn(blogDashboardBackendApiService, 'fetchBlogDashboardDataAsync')
         .and.returnValue(Promise.reject(500));
       spyOn(alertsService, 'addWarning');
 
-      component.ngOnInit();
+      component.initMainTab();
```

Comment:
Just to check, do you need an await here? (If so, why; if not, why not?)

Reply:
Also why did we need to change this?

Reply:
> Also why did we need to change this?

@seanlip I initially modified the code to manually invoke the initMainTab function. However, upon further reflection, I realized that such changes might not be necessary, and both versions of the code should be functioning correctly. Therefore, I reverted my modifications on that line, and the code continues to work as intended.

Reply:
Thanks, sg. I'm merging!